sechub-scan/src/main/resources/templates/report/html/scanresult.html

```diff
@@ -81,8 +81,51 @@
 			</table>
 		</div>
 		<div class="content">
+			<div>
+				<h2 class='summaryHeadline'>Summary</h2>
+				<table class='summaryTable'>
+					<thead>
+						<tr>
+							<th></th>
+							<th>Code Scan</th>
+							<th>Secret Scan</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr>
+							<td><strong>Total</strong></td>
+							<td><strong><span th:text="*{#lists.size(redList) + #lists.size(yellowList) + #lists.size(greenList)}">[Total number of Elements]</span></strong></td>
+							<td><strong></strong></td>
+						</tr>
+						<tr>
+							<td>Red</td>
+							<td>
+								<span th:if="!${redList.isEmpty()}"><a href="#redFindingsBlock" th:text="${#lists.size(redList)}"></a></span>
+								<span th:unless="!${redList.isEmpty()}" th:text="0"></span>
+							</td>
+							<td></td>
+						</tr>
+						<tr>
+							<td>Yellow</td>
+							<td>
+								<span th:if="!${yellowList.isEmpty()}"><a href="#yellowFindingsBlock" th:text="${#lists.size(yellowList)}"></a></span>
+								<span th:unless="!${yellowList.isEmpty()}" th:text="0"></span>
+							</td>
+							<td></td>
+						</tr>
+						<tr>
+							<td>Green</td>
+							<td>
+								<span th:if="!${greenList.isEmpty()}"><a href="#greenFindingsBlock" th:text="${#lists.size(greenList)}"></a></span>
+								<span th:unless="!${greenList.isEmpty()}" th:text="0"></span>
+							</td>
+							<td></td>
+						</tr>
+					</tbody>
+				</table>
+			</div>
 			<div th:if="!${redList.isEmpty()}">
-				<h2 class='redFindingHeadline'>Red findings</h2>
+				<h2 class='redFindingHeadline' id="redFindingsBlock">Red findings</h2>
```

Comment:
Each finding table needs a separate heading.
For example:

~~~
<h3 class='redFindingHeadline' id="codeScanRedFindingsBlock">Code Scan</a>
~~~